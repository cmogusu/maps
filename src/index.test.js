describe('testng promises',()=>{
	it('completes successfully',()=>{
		expect('true').toBe('true');
	})
})