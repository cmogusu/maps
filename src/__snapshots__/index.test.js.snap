// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots!! -  creates 1st snapshot 1`] = `
Array [
  <span
    className="absolute bottom left right z-index-1 opacity-8"
  >
    word of the day : 
    what up ma niggas
  </span>,
  <br />,
]
`;

exports[`Snapshots!! -  creates mutable snapshot 1`] = `
Object {
  "step1": "go to loo",
  "step2": "open door",
  "step3": "step inside",
  "step4": Any<String>,
  "step5": Any<Date>,
}
`;
